graphs/breadth_first_search.py

```diff
@@ -53,7 +53,7 @@ def bfs(self, start_vertex: int) -> set[int]:
         visited = set()
 
         # create a first in first out queue to store all the vertices for BFS
-        queue = Queue()
+        queue: Queue = Queue()
```

Comment:
Was mypy really failing this file just because of this annotation?!?

Reply:
It was, I was surprised too.